Gruntfile.js

```diff
@@ -37,9 +37,9 @@ module.exports = function(grunt) {
                 expand: true
             },
             build: {
-                cwd: './assets/css',
-                src: [ 'theme.css' ],
-                dest: './build/assets/css',
+                cwd: './assets/icomoon/fonts/',
```

Comment:
So, this copies the fonts but omits CSS, we want both, right? can you make copying font a separate grunt task instead?